dataset/src/main/openapi/dataset.yaml

```diff
@@ -263,8 +264,8 @@ paths:
       operationId: uploadTwingraph
       tags:
         - dataset
-      summary: Upload Twingraph with ZIP File
-      description: Upload Twingraph ZIP
+      summary: Upload data from zip file to dataset's twingraph
+      description: Upload data from zip containing two folder 'edges' and 'nodes' themself containing nodes and edges as csv files.
```

Comment:
Maybe copy the description in the twingraph/create_graph endpoint in this endpoint